docker-compose.yml

```diff
@@ -133,76 +126,10 @@ services:
       AC_WORLD_DATABASE_INFO: "ac-database;3306;root;${DOCKER_DB_ROOT_PASSWORD:-password};acore_world"
       AC_CHARACTER_DATABASE_INFO: "ac-database;3306;root;${DOCKER_DB_ROOT_PASSWORD:-password};acore_characters"
       AC_CLOSE_IDLE_CONNECTIONS: "0"
-    #profiles: [db-import]
-
-  ac-dev-database:
-    image: mysql:8.0
-    restart: unless-stopped
-    profiles: [dev]
-    cap_add:
-      - SYS_NICE # CAP_SYS_NICE
-    ports:
-      - ${DOCKER_DB_EXTERNAL_PORT:-63300}:3306
-      - ${DOCKER_WORLD_EXTERNAL_PORT:-8085}:8085
-      - ${DOCKER_SOAP_EXTERNAL_PORT:-7878}:7878
-      - ${DOCKER_AUTH_EXTERNAL_PORT:-3724}:3724
-    expose:
-      - 3306
-    environment:
-      - MYSQL_ROOT_PASSWORD=${DOCKER_DB_ROOT_PASSWORD:-password}
-    volumes:
-      - type: volume
-        source: ac-dev-database
-        target: /var/lib/mysql
-    healthcheck:
-      test: '/usr/bin/mysql --user=root --password=$$MYSQL_ROOT_PASSWORD --execute "SHOW DATABASES;"'
-      interval: 2s
-      timeout: 40s
-      retries: 20
-    networks:
-      ac-dev-network:
-        aliases:
-          - ac-database # needed by the server *.conf file
-
-  ac-dev-server:
-    tty: true
-    privileged: true
-    user: root
-    image: acore/ac-wotlk-dev-server:${DOCKER_IMAGE_TAG:-7.0.0-dev.1}
-    environment:
-      DATAPATH: "/azerothcore/env/dist/data"
-      DOCKER_ETC_FOLDER: "env/dist/etc"
-      AC_LOGIN_DATABASE_INFO: "ac-database;3306;root;${DOCKER_DB_ROOT_PASSWORD:-password};acore_auth"
-      AC_WORLD_DATABASE_INFO: "ac-database;3306;root;${DOCKER_DB_ROOT_PASSWORD:-password};acore_world"
-      AC_CHARACTER_DATABASE_INFO: "ac-database;3306;root;${DOCKER_DB_ROOT_PASSWORD:-password};acore_characters"
-      AC_DATA_DIR: "/azerothcore/env/dist/data"
-      AC_LOGS_DIR: "/azerothcore/env/dist/logs"
-    profiles: [dev]
-    volumes:
-      - ac-dev-server:/azerothcore
-      - ./var/shared:/azerothcore/var/shared
-    # network_mode: service:ac-dev-database
-    working_dir: /azerothcore
     depends_on:
-      ac-dev-database:
+      ac-database:
         condition: service_healthy
-    networks:
-      - ac-dev-network
 
-  ac-dev-tools:
-    <<: *ac-shared-conf
-    image: acore/ac-wotlk-tools:${DOCKER_IMAGE_TAG:-7.0.0-dev.1}
-    working_dir: /azerothcore/env/client/
-    cap_add:
-      - SYS_NICE # CAP_SYS_NICE
-    volumes:
-      - ${DOCKER_CLIENT_DATA_FOLDER:-./var/client}:/azerothcore/env/client/Data
-      - ${DOCKER_VOL_TOOLS_CAMERAS:-ac-client-data-cameras}:/azerothcore/env/dist/data/Cameras
-      - ${DOCKER_VOL_TOOLS_DBC:-ac-client-data-dbc}:/azerothcore/env/client/dbc
-      - ${DOCKER_VOL_TOOLS_MAPS:-ac-client-data-maps}:/azerothcore/env/client/maps
-      - ${DOCKER_VOL_TOOLS_VMAPS:-ac-client-data-vmaps}:/azerothcore/env/client/vmaps
-      - ${DOCKER_VOL_TOOLS_MMAPS:-ac-client-data-mmaps}:/azerothcore/env/client/mmaps
-    profiles: [dev]
```

Comment:
why are you removing all these ac-dev*?

Reply:
I noted in the PR's body that containers related to development have been removed in favor of those in `azerothcore-wotlk`

Having 2 repositories organized for development on azerothcore makes it more difficult to provide support for both. Because of that, I think it's in our best interest to tighten the scope of this repository as much as possible.